src/engraving/libmscore/spanner.cpp

```diff
@@ -385,6 +385,11 @@ Spanner::Spanner(const Spanner& s)
     _tick         = s._tick;
     _ticks        = s._ticks;
     _track2       = s._track2;
+    if (!s.startElement() && !spannerSegments().size()) {
```

Comment:
logically I'd expect spanner segments to always be cloned, but it was causing some issues for me, hence for now I've only put it in for when the element is cloned when pasting from clipboard